### ISSUE TITLE
Some ground station parameters are unclear

### ISSUE THREAD

**Issue by sh123** [QDeltaSoft/qradiopredict#11]
Hello, 

was trying qradiopredict and some station parameters need clarification. Maybe they need drop down lists instead of input controls?

For example:
"Transmission type", "Polarization", "Tx antenna type", "Rx antenna type"

Document does not state what are acceptable values, not clear from the UI what to specify there.

E.g. I tried simulations with "Polarization" set to 1, is it horizontal polarization or vertical?

**Comment by kantooon** [comment]
Yes, I agree the whole UI needs improvement. Some time will be dedicated to this in the near future.
